src/assets/config_files/usecases.json

```diff
@@ -115,17 +115,51 @@
     ],
     "subcellular": [
       {
-        "title": "KappaNeuron how to",
-        "description": "An example use case on how to use KappaNeuron",
+        "title": "KappaNEURON demonstration",
+        "description": "A simple example use case of how to use KappaNEURON for a calcium pump",
         "experience": ["all"],
         "maturity": ["beta"],
         "disabled": true,
         "picture": {
-          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/placeholder.jpg"
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/KappaNEURON.png"
+          "alt": "Thumbnail of KappaNEURON traces"
```

Comment:
There is a comma missing on the previous line that is why this is red, it's malformed